docs/semgrep-app/notifications.md

```diff
@@ -277,31 +277,158 @@ Automated comments on GitLab merge requests are displayed as follows:
+:::
+
+Create a repository access token in Bitbucket. Follow the instructions in [Create a repository Access Token](https://support.atlassian.com/bitbucket-cloud/docs/create-a-repository-access-token/). Alternatively, follow these steps to create a repository access token:
+
+1. Sign in to [Semgrep Cloud Platform](https://semgrep.dev/login). See [Signing in to Semgrep Cloud Platform](/semgrep-app/getting-started-with-semgrep-app/#signing-in-to-semgrep-app) for details on requested repository permissions and access.
+1. Click **[Projects](https://semgrep.dev/orgs/-/projects)** > **Scan New Project** > **Run Scan in CI**.
+1. Select Bitbucket from the menu.
+1. Click **Create new API token**, and then copy the value of the new API token.
+1. In Bitbucket, select the repository where you want Semgrep to run in CI.
+1. Click **Repository settings** > **Repository variables**.
+1. Create a new repository variable by entering **SEMGREP_APP_TOKEN** into the **Name** field, and then paste the value of the token in the **Value** field.
+1. Enable the **Secured** option, and then click **Add**.
+1. Click **Repository settings** > **Access tokens**.
+1. Click **Create Repository Access Token**.
+1. Create a token name. Note: As Semgrep comments to your PRs, use **Semgrep** as the name for this token.
+1. Under **Scopes**, select the **Read**, and **Write** permissions for pull requests.
+1. Click **Create**.
+1. Copy the value of the repository access token, and then click **Close**.
+1. Click **Repository variables**.
+1. Create a new repository variable by entering **PAT** (change this placeholder name as necessary) to the **Name** field, and then paste the workspace access token in the **Value** field.
+1. Enable the **Secured** option, and then click **Add**.
+
+To complete the configuration, follow the [Adding Semgrep to your Bitbucket CI pipeline for PR comments](#adding-semgrep-to-your-bitbucket-ci-pipeline-for-pr-comments) section.
+
+#### Adding Semgrep to your Bitbucket CI pipeline for PR comments
+
+To receive comments on PRs, commit the following code to your `bitbucket-pipelines.yml` file:
+
+```yaml
+image: atlassian/default-image:latest
+
+pipelines:
+  branches:
+    main:
+        - step:
+            name: 'Run Semgrep full scan with main branch'
+            image: returntocorp/semgrep
+            script:
+              - export SEMGREP_APP_TOKEN=$SEMGREP_APP_TOKEN
+              - semgrep ci
+  pull-requests:
+    '**':
+      - step:
+          name: 'Run Semgrep diff scan with PR branch'
+          image: returntocorp/semgrep
+          script:
+            # Set SEMGREP Variables
+            - export SEMGREP_APP_TOKEN=$SEMGREP_APP_TOKEN
+            - export SEMGREP_BASELINE_REF="origin/main"
+            - git fetch origin "+refs/heads/*:refs/remotes/origin/*"
+            - export BITBUCKET_TOKEN=$PAT
+            - semgrep ci
+```
```

Comment:
@s-santillan just tagging you here, I was not sure if you want to add this to `/src/components/code_snippets/_bitbucket-semgrep-app-sast.mdx` and add more comments. :)